recipes/quill/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

CXX_STANDARD is cmake 3.0